LayoutKit/Layouts/OverlayLayout.swift

```diff
@@ -0,0 +1,76 @@
+// Copyright 2017 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import CoreGraphics
+
+/**
+ A layout that overlays others. Allows adding other layouts behind or above a primary layout.
+ The size of the primary, background, and overlay layouts will be determined based on the size
+ computed from the primary layout.
+ */
+open class OverlayLayout<V: View>: BaseLayout<V> {
+
+    open let primary: Layout
+    open let background: [Layout]
+    open let overlay: [Layout]
+
+    public init(primary: Layout,
```

Comment:
Could you also add some comments here? Especially for `primary`, `background` and `overlay`.